pkg/network/config/config.go

```diff
@@ -262,5 +266,15 @@ func New() *Config {
 		}
 	}
 
+	// For backwards compatibility, check to see if system_probe.windows.driver_buffer_size differs
+	// from its default (indicating it got set by the user). If so, use it in place of
+	// network_config.windows.driver_buffer_size
+	deprecatedDriverBufferSize := cfg.GetInt(join(spNS, "windows.driver_buffer_size"))
+	if deprecatedDriverBufferSize != ddconfig.DefaultDriverBufferSize {
```

Comment:
There is a probably a cleaner way to do this. If you don't set a default value for either setting, by using `BindEnv` rather than `BindEnvAndSetDefault`, then you can use `if cfg.IsSet(...)` here. That should simplify this logic to:
```
if cfg.IsSet(join(netNS, "windows.driver_buffer_entries")) {
  // use set value and only set DriverBufferEntries
} else if cfg.IsSet(join(spNS, "windows.driver_buffer_size)) {
  // use set value and only set DriverBufferSize
} else {
  // set DriverBufferEntries to DefaultFlowEntries
}
```
Later on you can do
```
if cfg.DriverBufferSize > 0 {
  // use DriverBufferSize
} else {
  // use DriverBufferEntries
}
```

Reply:
That is a lot cleaner, thank you for the suggestion!